flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/stream/sql/JoinTest.scala

```diff
@@ -1003,12 +980,43 @@ class JoinTest extends TableTestBase {
     util.verifyTable(result, expected)
   }
 
+  private def verifyTimeBoundary(
```

Comment:
Please either revert this move or extract it to separate commit. As it is now, I can not easily tell if something has changed or if that's a simple refactor without any functional change.